src/audio/copier/copier.c

```diff
@@ -950,8 +950,8 @@ static int copier_comp_trigger(struct comp_dev *dev, int cmd)
 
 	dai_copier = pipeline_get_dai_comp_latency(dev->pipeline->pipeline_id, &latency);
 	if (!dai_copier) {
-		comp_err(dev, "failed to find dai comp or sink pipeline not running.");
-		return ret;
+		comp_warn(dev, "failed to find dai comp or sink pipeline not running.");
+		return 0;
```

Comment:
Ok, so this looks suspect without an inline comment. Can you add one.

Reply:
@lyakh, would you add a patch to your #7210 to change the return value to some error code as  a last step? This should not happen now afaik, so if there is no copier found it is an error?
Then I can close this PR.